openssl/src/pkcs7.rs

```diff
@@ -146,6 +183,82 @@ impl Pkcs7 {
             .map(Pkcs7)
         }
     }
+
+    /// Set signed attributes in a PKCS#7 structure.
+    ///
+    /// `attributes` is a stack of the attributes to be added.
+    ///
+    #[corresponds(PKCS7_set_signed_attributes)]
+    pub fn set_signed_attributes(
+        signer_info: &Pkcs7SignerInfoRef,
+        attributes: Stack<X509Attribute>,
+    ) -> Result<(), ErrorStack> {
+        unsafe {
+            cvt(ffi::PKCS7_set_signed_attributes(
+                signer_info.as_ptr(),
+                attributes.as_ptr(),
+            ))?;
+            mem::forget(attributes);
```

Comment:
`PKCS7_set_signed_attributes` does not take ownership of the stack - this function should take `&StackRef<X509Attribute>`.

Reply:
Fixed.